settings.py

```diff
@@ -115,4 +119,20 @@ def get_listen_ip(self):
     def get_listen_port(self):
         return self['listen'].split(':')[1]
 
+    def check_user(self, user, pswd):
+        try:
+            au = self['user']
+            pw = self['password']
+            logging.info("check_user(" + user + "," + pswd + "), looking for (" + au + "," + pw + ")")
+            if au == "" and pw == "":
```

Comment:
No, "not au and not pw" gives a different answer if either were a value that evaluated as false. Of course, if everything is working properly, au and pw are strings and your comment is correct. I was being more conservative.